config/app.php

```diff
@@ -55,6 +55,26 @@
     'url' => env('APP_URL', 'http://localhost'),
     'mix_url' => env('APP_URL', 'http://localhost'),
 
+     /*
+    |--------------------------------------------------------------------------
+    | Branding
+    |--------------------------------------------------------------------------
+    |
+    | Configuration used for branding purposes of this website.
+    |
+    */
+
+    'title' => env('SITE_TITLE', 'Booking'),
+    'division' => env('SITE_DIVISION', 'Unknown Division'),
+    'division_url' => env('SITE_DIVISION_URL', '#'),
+    'contact_mail' => env('SITE_CONTACT_MAIL'),
+
+    'pilotbrief_url' => env('SITE_PILOTBRIEF_URL'),
+    'charts_url' => env('SITE_CHARTS_URL'),
+
+    'color_navbar' => env('SITE_COLOR_NAVBAR'),
+    'color_links' => env('SITE_COLOR_LINKS'),
```

Comment:
Can you please add all fallback values? I'm actually not sure if there's more people using BMAC already, but people might forget the ``.env`` file

Reply:
I added those in .env-example, but you're right people might forget. So I added it as fallback as well.